lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -334,9 +445,11 @@ void SessionStateEnforcer::m5g_remove_rules_and_update_upf(
  * seid - session context id to find respective session

```

Comment:
Should this be `teid`? Is this doc still up to date?

Reply:
right --   while the response coming from UPF, we use teid to locate the session. while session request/modification coming from AMF, we use "pdu_id". There is a small change we have done recently as per AMF needs.